fact_helper_file/mime/custom_mime_firmware_containers

```diff
@@ -714,3 +714,13 @@
 # SDN unpacker (artificial)
 0	string \x53\x44\x4E	FKIE SDN Unpacker
 !:mime	firmware/sdn
+
+# Draytek Vigor Firmware
+# Source: https://github.com/q3k/vraytekdigor/blob/master/build/fwtool/fwimage.go
+0       string  2RDH    Draytek Vigor Firmware Container
+!:mime firmware/draytek
+>4      ubelong x       header size: %d bytes,
+>8      ubelong x       file size: %d bytes,
+>12     ubelong x       crc32: 0x%X,
+>80     ubelong x       kernel size: %d bytes,
+>84     ubelong x       squashfs size: %d bytes,
```

Comment:
🤔 Something's not quite right here. Those are all unsigned integers, shouldn't you use a `%u` format string placeholder then?